packages/mds-transaction-service/tests/transactions.spec.ts

```diff
@@ -42,6 +44,7 @@ describe('Transaction Repository Tests', () => {
 const TransactionServer = TransactionServiceManager.controller()
 
 describe('Transaction Service Tests', () => {
+  let mockStream = stream.mockStream(TransactionStreamKafka)
```

Comment:
Can this be a `const` instead of a `let`, and initialized here?

Reply:
It needs to be reinitialized for each test or it won't count invocations of write properly, I believe.

Reply:
Yes, we tried it the other way.

Reply:
Ah, I see. If you simply need to reset the state of the mock after each test, you should be able to call `jest.resetAllMocks()` inside of the `beforeEach`, as opposed to re-creating it from scratch each time.

Reply:
That doesn't seem to work. Maybe you could help take a look?